templates/common/.devcontainer/Dockerfile/base/Dockerfile

```diff
@@ -3,4 +3,3 @@ FROM --platform=amd64 mcr.microsoft.com/devcontainers/${IMAGE}
 RUN export DEBIAN_FRONTEND=noninteractive \
      && apt-get update && apt-get install -y xdg-utils \
      && apt-get clean -y && rm -rf /var/lib/apt/lists/*
-RUN curl -fsSL https://aka.ms/install-azd.sh | bash
```

Comment:
I think we should be able to delete the custom Dockerfiles now.  Our custom feature already installs `xdg-utils` which is all this was providing for us on top of installing `azd`, so I think the whole thing can be removed and we can just start referencing standard devcontainer images instead of having to build a custom container starting from the base images.

Reply:
That probably makes sense for the `base` Dockerfile, but the Java/Func ones are a bit tricker (although there is a [Functions Core Tools feature](https://github.com/jlaundry/devcontainer-features/tree/main/src/azure-functions-core-tools) we could use).

But I'm unsure how the `devcontainer.json` knows about the `Dockerfile` that it's using, since they aren't collocated, and I can't figure it out from the repo.

Reply:
Can we get that func core tools devcontainer feature moved to official?  I'm hesitant to take dependency on personal dev feature.

Might need to chat with Paul about that.

Reply:
That's a fair not @jongio - let us know what the Functions team says, I can always create a Feature for them if they don't have the bandwidth (although I am hesitant to "trump" the work of the community).

Reply:
Can we reach out to the community person and see if they want to make it official...kind of like what we did with this one...